_posts/en/2020-07-28-micro-procurment-whats-stopping-us.md

```diff
@@ -0,0 +1,107 @@
+
+Instead of the example above regarding finding and fixing a bug, imagine that you need a small IT solution.  You find an open source application that would meet your needs but it does
+not meet GC accessibility requirements. What if you could quickly and easily pay someone to fix the accessibility issues so that you could choose that open source solution and move
+forward without delay?  If this existed would you be more likely to consider using open source solutions?
+
+Private industry has been doing low dollar value purchases of code, including [bug bounty initiatives](https://www.itworldcanada.com/article/bug-bounty-programs-growing-stronger/413203)
+for some time.  Other governments have been working on this as well. The BC government has had a great deal of success with the [BCDevExchange](https://bcdevexchange.org/) 'Code with Us'
+program. This program has since expanded in scope (to contracts up to $70K) and evolved into the [Digital Marketplace](https://digital.gov.bc.ca/marketplace) platform.
+
+The US gov has a [simplified process for all micro-purchases](https://acquisition.gov/content/part-13-simplified-acquisition-procedures#i1111868) which, among other things, encourages
+their Deputy Heads to delegate authority for micro-purchasing. Many groups within the US government have experimented with using this process to purchase open source code including the
+[IT Modernization CoE](https://digital.gov/2020/04/24/experimenting-with-micropurchase-threshold/), the [Department of Veterans' Affairs](https://github.com/
+department-of-veterans-affairs/VA-Micropurchase-Repo), and [18F](https://github.com/18f/micropurchase-archive).
+
+The Government of Canada has already dipped its toe into micro-procurement for open source code with the 2017 launch of the pilot [GC Developers Exchange](https://github.com/canada-ca/
+devex) by Privy Council Office (in partnership with TBS and PSPC). All contracts awarded via GCDevEx were $10K and under, and were for virtual and unclassified work only (eliminating the
+need for a security check). The GCDevEx pilot created many [lessons learned](https://github.com/canada-ca/devex/issues/51), but ultimately went offline as it did not include simplified
+back-end procurement processes and it struggled to find a new lead following the completion of the pilot by PCO.
+
+At this point it is important to be clear that micro-procurement is not about increasing purchasing or outsourcing. It's about doing purchasing in a different way. With the simplified
+process that comes with micro-procurement, you get the thing you need faster.  When the micro-procurement is available for open source code, it provides an ad-hoc method of support and
+could, as a result, increase the uptake of open source use in the GC (the use of which is a [Digital Standard](https://www.canada.ca/en/government/system/digital-government/
+government-canada-digital-standards.html))
```

Comment:
And would align us with the [Digital Nations Charter](https://www.canada.ca/en/government/system/digital-government/improving-digital-services/digital9charter.html), of which Canada is a signatory

Reply:
Added.